### ISSUE TITLE
fundrawtransaction.py test broken in travis ("Mempool sync failed")

### ISSUE THREAD

**Issue by laanwj** [bitcoin/bitcoin#9364]
When it fails (seems intermittent but with high frequency) it fails consistently with the same exception: "Mempool sync failed".

However where exactly it happens seems random, though it's always triggered by a `sync_all()`.

- in https://travis-ci.org/bitcoin/bitcoin/jobs/184431201#L3798 it's the sync_all on line 534 that fails
- in https://travis-ci.org/bitcoin/bitcoin/jobs/184473580#L2953 it's the one on line 461
- in https://travis-ci.org/bitcoin/bitcoin/jobs/184333528#L2919 it's the one on line 449
- in https://travis-ci.org/bitcoin/bitcoin/jobs/184328060#L3450 it also happens on line 534

I cannot reproduce this locally.

**Comment by laanwj** [comment]
This has likely been triggered by one of yesterday's merges:
```
    Merge #9322: [qa] Don't set unknown rpcserialversion
    Merge #9313: If we don't allow free txs, always send a fee filter
    Merge #9331: [qa] Add test for rescan feature of wallet key import RPCs
    Merge #7562: Bump transaction version default to 2
    Merge #9172: Resurrect pstratem's "Simple fuzzing framework"
```

**Comment by laanwj** [comment]
Fixed by #9367
